release notes/v0.47.0.md

```diff
@@ -59,6 +59,10 @@ context.clearCookies();
 console.log(context.cookies.length); // 0
 ```
 
+### Add support for browser module's `page.on('console')` [browser#1006](https://github.com/grafana/xk6-browser/pull/1006)
```

Comment:
Might want to reference https://github.com/grafana/xk6-browser/pull/1010?

Reply:
Should this be included in bug fixes instead maybe? It might be confusing to associate this PR directly with this feature as its description does not reference it directly, even if in practice it was :thinking:

Reply:
Yeah sure, happy for it to be part of bug fixes or internal improvements 👍